include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -259,23 +259,27 @@ ERROR(failed_base_method_call_synthesis,none,
       (ValueDecl *, ValueDecl *))
 
 ERROR(both_returns_retained_returns_unretained, none,
-      "%0 cannot be annotated with both swift_attr('returns_retained') and "
-      "swift_attr('returns_unretained') attributes",
+      "%0 cannot be annotated with both 'SWIFT_RETURNS_RETAINED' and "
+      "'SWIFT_RETURNS_UNRETAINED'",
       (const clang::NamedDecl *))
 
 ERROR(returns_retained_or_returns_unretained_for_non_cxx_frt_values, none,
-      "%0 cannot be annotated with either swift_attr('returns_retained') or "
-      "swift_attr('returns_unretained') attribute because it is not returning "
+      "%0 cannot be annotated with either 'SWIFT_RETURNS_RETAINED' or "
+      "'SWIFT_RETURNS_UNRETAINED' because it is not returning "
       "a 'SWIFT_SHARED_REFERENCE' type",
       (const clang::NamedDecl *))
 
-// TODO: make this case an error in next cxx-interop versions rdar://138806722
-WARNING(
-    no_returns_retained_returns_unretained, none,
-    "%0 is returning a 'SWIFT_SHARED_REFERENCE' type but is not annotated "
-    "with either swift_attr('returns_retained') or "
-    "swift_attr('returns_unretained') attributes",
-    (const clang::NamedDecl *))
+WARNING(no_returns_retained_returns_unretained_warn, none,
+        "%0 would not be imported in a later version of Swift because it is "
```

Comment:
I think we can use simpler grammar and just say "is not imported in a later version"